openmdao/recorders/base_case_reader.py

```diff
@@ -0,0 +1,44 @@
+"""
+Docstring.
+"""
+from abc import ABCMeta
+
+
+class BaseCaseReader(object):
+    """
+    Abstract base class of all CaseReader implementations.
+
+    Parameters
+    ----------
+    filename : str
+        The name of the file from which to instantiate the case reader.
+
+    Attributes
+    ----------
+    format_version : int
+        An integer representation of the format version in the recorded file.
+    filename : str
+        The name of the file from which the recorded cases are to be loaded.
+    parameters : dict
```

Comment:
These lasthree attributes don't match the ones in initialize. Also, missing 5 others.